ci/pipelines/deploy-to-perf.yml

```diff
@@ -27,6 +27,35 @@ definitions:
     aws_ecr_registry_id: "((pay_aws_test_account_id))"
     aws_region: eu-west-1
 
+  put_db_migration_slack_notification: &put_db_migration_slack_notification
+    put: slack-notification
+    params:
+      channel: '#govuk-pay-announce'
```

Comment:
I put the success in #govuk-pay-activity because Kat indicated [only the failures should go into #govuk-pay-announce](https://github.com/alphagov/pay-ci/pull/679#pullrequestreview-960147018).

Reply:
Fair enough, so how about putting the start notice into -activity too?

Reply:
Fixed